.gitignore

```diff
@@ -3,3 +3,4 @@
 pubspec.lock
 test/.test_cov.dart
 coverage/
+/test/widget_test.dart
```

Comment:
Doesn't seem to be used.

Reply:
This was me trying to learn AndroidStudio (click this, click that, suddenly files show up) to run the tests.